dragonflybsd/README.md

```diff
@@ -0,0 +1,137 @@
+# DragonflyBSD fwupd port notes
+
+## Preparation
+
+- Installed packages
+    - qemu-system-x86-64
+    - ovmf
+- Downloaded
+[latest version](https://mirror-master.dragonflybsd.org/iso-images/?C=M;O=D) of
+DragonflyBSD release in uncompressed `.iso` format
+- Created Virtual disk with `qemu-img` command:
+
+```
+$ qemu-img create -f qcow2 disk.qcow2 15G
+```
```

Comment:
This command is duplicated in the installation section, we could link that section instead, because someone could run it twice